frontend/templates/frontend/download_project.html

```diff
@@ -19,9 +76,12 @@
         refresh_availabilities();
     });
 </script>
-{% endblock %} {% block content %}
+
 <form action="/download_project/" method="post" id="calcform" enctype="multipart/form-data">
+    <div class="formContainer">
     {% csrf_token %}
+    <h1 id="downloadHeading">Download Project</h1>
+    <h3 id="project_heading">Project Heading</h3>
```

Comment:
You should add the name of the project here instead of "Project Heading"

Reply:
Ohh, sorry I forgot to add it.

Reply:
Could you quickly fix, commit and push it? It will update the PR automatically.

Reply:
I have pushed the changes.